### DISCUSSION TITLE
Dependent not listed in Dependency graph

### DISCUSSION THREAD

**Discussion by phseiff** [community/community#21634] [New to GitHub]
<p>Hello,</p>
<p>I’m having the same issue with the dependency graph as the one <a href="https://github.community/t/dependency-graph-showing-dependencies-but-not-dependents/3315">described in this help thread</a>:<br>
One of my repositories (phseiff/github-flavored-markdown-to-html) has a dependent (theendlessriver13/gh-md-to-moodle). This dependent also lists my repository in its dependencies in the dependency graph, but my project does not list it in its dependents for some reason.</p>
<p>So A lists B as a dependency in GitHub’s dependency graph, but B doesn’t list A as a dependent in the dependency graph, which definitely differs from GitHub’s intended behavior.<br>
So just to be clear, this is an issue about the dependency graph not behaving as expected, and NOT about these specific projects (which I know would be off-topic).</p>
<p>Would it be possible to provide manual assistance in this issue, like you did for <a href="https://github.community/t/dependency-graph-showing-dependencies-but-not-dependents/3315">this issue</a>?</p>
<p>Thanks in advance!</p>

**Comment by ernest-phillips** [comment]
<aside class="quote no-group" data-username="phseiff" data-post="1" data-topic="174456">
<div class="title">
<div class="quote-controls"></div>
 phseiff:</div>
<blockquote>
<p>theendlessriver13/gh-md-to-moodle</p>
</blockquote>
</aside>
<p>Hi :wave:</p>
<p>Happy to investigate this. It is likely that your package is not associated with the correct repository.<br>
Is this your package? <a href="https://pypi.org/project/markdown-to-html/" class="inline-onebox">markdown-to-html · PyPI</a></p>
<p>If not, could you please point me to the pypi page link?</p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Hi <a class="mention" href="/phseiff">@phseiff</a> ,</p>
<p>It looks like we needed to redo the assignment of this package to your repository.</p>
<p>Your dependents are currently showing</p><aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/phseiff/github-flavored-markdown-to-html/network/dependents" target="_blank" rel="noopener">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/5134583/181065927-d307b90d-5035-4dfc-aa19-b8400c77a968.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/2/32f87bfb33a4854e28bca501497c39518dd9fe4c_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/3/2/32f87bfb33a4854e28bca501497c39518dd9fe4c_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/3/2/32f87bfb33a4854e28bca501497c39518dd9fe4c.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/2/32f87bfb33a4854e28bca501497c39518dd9fe4c_2_10x10.png"></div>

<h3><a href="https://github.com/phseiff/github-flavored-markdown-to-html/network/dependents" target="_blank" rel="noopener">phseiff/github-flavored-markdown-to-html</a></h3>

<p><a href="https://github.com/phseiff/github-flavored-markdown-to-html/network/dependents" target="_blank" rel="noopener">rk/dependents</a></p>

  <p><span class="label1">Convert markdown to HTML using the GitHub API and some additional tweaks with Python. Comes with full formula support and image compression.</span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by phseiff** [comment]
<blockquote>
<p>Is this your package? <a href="https://pypi.org/project/markdown-to-html/" rel="noopener nofollow ugc">markdown-to-html · PyPI</a></p>
</blockquote>
<p>It’s <a href="https://pypi.org/project/gh-md-to-html/" rel="noopener nofollow ugc">gh-md-to-html</a>, actually :sweat_smile:</p>
<p>But it works now, so I assume you found the right one?</p>
<p>Thank you very much for solving this! :smiley:</p>

**Comment by ernest-phillips** [comment]
<p>Yep, I landed on that other one first but went back and found the right package associated with your repo. Thanks for your patience here.</p>

**Comment by fabiocaccamo** [comment]
Hi @ernest-phillips 

I'm experiencing the same issue with all my Python / Django packages after I switched from `setup.py` to `setup.cfg` (and I have also a `pyproject.toml` file, but only for defining `Black` settings):

- https://github.com/fabiocaccamo/django-admin-interface

- https://github.com/fabiocaccamo/python-benedict

- https://github.com/fabiocaccamo/django-colorfield

- https://github.com/fabiocaccamo/django-treenode

- https://github.com/fabiocaccamo/django-maintenance-mode

- https://github.com/fabiocaccamo/django-extra-settings

- https://github.com/fabiocaccamo/python-fsutil

- https://github.com/fabiocaccamo/django-freeze

- https://github.com/fabiocaccamo/python-codicefiscale

- https://github.com/fabiocaccamo/django-redirects

- https://github.com/fabiocaccamo/python-fontbro
